### ISSUE TITLE
PURE - new wallet 1.0.0.3 pure.conf changes

### ISSUE THREAD

**Issue by hashtafak** [masternodes/vps#89]
OLD config
bind=IP:PORT (not work)

NEW config
port=PORT (no need/not sure)
masternode=1
masternodeaddr=IP:PORT
masternodeprivkey=PRIVKEY

u guys need to change this to make new wallet v1.0.0.3 running

**Comment by naylordl** [comment]
Until its fixed completely I have a workaround. Before you install pure with the script, go into the config->pure folder and find the environment file for PURE called "pure.env". Change the version in there to v1.0.0.3 and then do your install. I did this and it works fine for me.

**Comment by hashtafak** [comment]
Yes u also can install with command line ./install.sh -p pure -r "tags/v1.0.0.3" which 1.0.0.3 is the lastest version of pure wallet

But it didn't work for me when i'm trying to start pured. The problem come from pure.conf because they changed some config key

**Comment by marsmensch** [comment]
Fixed by adding a dedicated config template (you guys can also do this for any required changes, just place it into the corresponding config directory) and bumped the version.

Ref: https://github.com/masternodes/vps/commit/a4525992ecaee8a8901a665749cf7e58690b5392
